tasks/section5.yml

```diff
@@ -530,6 +530,7 @@
 - name: "SCORED | 5.4.2 | PATCH | Ensure system accounts are non-login"
   shell: "awk -F: '($3 < {{ rhel7cis_rule_5_4_2_min_uid }}) {print $1 }' /etc/passwd"
   changed_when: no
+  checkmode: false
```

Comment:
If this awk does not make any changes, the title above should indicate audit vs patch.

Reply:
Also, check_mode.

Reply:
> If this awk does not make any changes, the title above should indicate audit vs patch.

I'm not too sure, as this task is half of a patch action.